src/main/java/com/minecolonies/coremod/entity/ai/mobs/Barbarians/AbstractEntityBarbarian.java

```diff
@@ -0,0 +1,180 @@
+package com.minecolonies.coremod.entity.ai.mobs.barbarians;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "AbstractEntityBarbarian.java" should be located in "com/minecolonies/coremod/entity/ai/mobs/barbarians" directory, not in "/home/marvin/Teamcity/buildAgent4/work/70f6e69b0ad5fca1/src/main/java/com/minecolonies/coremod/entity/ai/mobs/Barbarians". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1598)